BZ#1838164: prereqs for multiple interfaces (#29873)
openshift/openshift-docs: install/prerequisites.adoc

on must be in the default zone.
====

[[hosts-multiple-network-interfaces]]
==== Hosts with Multiple Network Interfaces

> If a host has more then one network interface, {product-title} uses only one network interface for installation, the cluster network, and the service network. Additional network interfaces could be used for communication that is not related to {product-title}, but there is no support to route some cluster-related traffic over one network interface and different cluster-related traffic over another network interface.

Reviewer: ```suggestion
If a host has more then one network interface, {product-title} uses only one network interface for installation, the cluster network, and the service network. You can use additional network interfaces for communication that is not related to {product-title}, but there is no support to route some cluster-related traffic over one network interface and different cluster-related traffic over another network interface.
```

> ✔️  can verb
Use to indicate ability. Use instead of "could". To indicate possibility, use "might".

Author: Let me revise to sound less encouraging of using that additional network interface.  Sorry for leaving "could" in there, I should have spotted that.
